### ISSUE TITLE
Mod Incompatibility: ItemScroller (Game hangs indefinitely when joining any world)

### ISSUE THREAD

**Issue by RodneyMKay** [FlorianMichael/ViaFabricPlus#102]
### General Information

**Environment:**
Java: 17
Fabric: v0.14.19

**Installed Mods:**
[FabricAPI](https://modrinth.com/mod/fabric-api) v0.81.1 [1.19.4]
[ItemScroller](https://www.curseforge.com/minecraft/mc-mods/item-scroller) v0.19.2 [1.19.4] _(This is the mod causing the incompatibility)_
[MaLiLib](https://www.curseforge.com/minecraft/mc-mods/malilib) v0.15.4 [1.19.4] _(Library mod required for ItemScroller)_
[ViaFabricPlus](https://modrinth.com/mod/viafabricplus) v2.7.1

All of the above are the latest version at the time of writing

### Bug Description

There seems to be some sort of incompatibility between the Mixins of ItemScroller and ViaFabricPlus that causes errors when joining worlds. I attached a full log of an attempt of me starting the game and trying to join a singleplayer world.

[2023-05-19-7.log](https://github.com/FlorianMichael/ViaFabricPlus/files/11516039/2023-05-19-7.log)

I believe the crucial piece of information is the following (though I'm not entirely certain):
```
[12:19:37] [Render thread/WARN]: @Redirect conflict. Skipping viafabricplus.mixins.json:fixes.minecraft.MixinClientPlayerInteractionManager from mod viafabricplus->@Redirect::modifySlotClickPacket(Lnet/minecraft/class_634;Lnet/minecraft/class_2596;)V with priority 1000, already redirected by mixins.itemscroller.json:MixinClientPlayerInteractionManager from mod itemscroller->@Redirect::bufferClickPacketsAndCancel(Lnet/minecraft/class_634;Lnet/minecraft/class_2596;)V with priority 1000
```
Referencing:
[Mixin in ItemScroller Code](https://github.com/maruohon/itemscroller/blob/f1d4b9e50cc4220ad338331d887056040fdc6472/src/main/java/fi/dy/masa/itemscroller/mixin/MixinClientPlayerInteractionManager.java#L27)
[Mixin in ViaFabric Code](https://github.com/FlorianMichael/ViaFabricPlus/blob/fcff14dc44facd13807997cd7ce848acc1f8ea71/src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/MixinClientPlayerInteractionManager.java#L106)

### Steps to Reproduce

The noticable ingame issue occurs when trying to join any world regardless of whether it's singleplayer or multiplayer. The game freezes and locks up. In multiplayer this happens in the "Joining world..." stage and in singleplayer it happens when the world loading animation reaches 100%.

### Expected Behavior

The game shouldn't hang.

PS: 
Thanks so much for creating this mod. It's truly amazing ❤
If any testing needs to be done, feel free to shoot me a message.

**Comment by FlorianMichael** [comment]
https://github.com/FlorianMichael/ViaFabricPlus/commit/5a7147eaaff8125968f16a6fd191c35c086585cf already fixed, will be published in the 2.7.2 version

**Comment by FlorianMichael** [comment]
https://github.com/FlorianMichael/ViaFabricPlus/releases/tag/v2.7.3
